### ISSUE TITLE
Transparency only works with window titlebar

### ISSUE THREAD

**Issue by xamddewey** [Luwx/Lightly#108]
My operating system is Manjaro with proprietary nvidia driver installed, I followed the README.md, but toolbar and sidebar still remain opaque...

**Comment by scratch-er** [comment]
Switch to wayland session to resolve this problem. This issues also comes to Kvantum themes on plasma 5.21 or later, transparency only works on wayland, maybe due to an API change.

**Comment by second2050** [comment]
I don't think this is an X11 problem as it works on my machine running the proprietary nvidia driver in X11, unless I'm missing something in this issue right now. 

![Screenshot_20210710_142116](https://user-images.githubusercontent.com/37344371/125162795-22c4ee80-e18a-11eb-943b-44dfdc36ed55.png)

**Comment by getaceres** [comment]
With KDE 5.23 in Kubuntu 21.10 I can confirm that transparency only works with Wayland but not X11.
I have an AMD Ryzen Pro 2700U.

**Comment by joickle** [comment]
I'm also having the same issue on Manjaro with KDE 5.23.3, Only the titlebar is transparent.

**Comment by joickle** [comment]
Figured out my issue, turns out the dev blocked it for a past issue. Not sure why it's still blocked, I removed the modification made by commit 39700ce and rebuilt. All works perfectly now with none of the scaling issues mentioned. :)
